onward/app/controllers/TaggedContentController.scala

```diff
@@ -54,12 +54,12 @@ class TaggedContentController(
     "theguardian/series/guardiancommentcartoon",
   )
 
-  private def lookup(tag: String, edition: Edition)(implicit request: RequestHeader): Future[List[ContentType]] = {
-    log.info(s"Fetching tagged stories for edition ${edition.id}")
+  private def lookup(tag: String)(implicit request: RequestHeader): Future[List[ContentType]] = {
+    log.info(s"Fetching tagged stories")
     contentApiClient
       .getResponse(
         contentApiClient
-          .search(edition)
```

Comment:
This change concerns me ... I want to look into what it was doing / why the edition was considered to matter

Reply:
I've looked into this - I don't think this is used anywhere https://github.com/guardian/frontend/issues/25606